qtodotxt/ui/controllers/menu_controller.py

```diff
@@ -23,13 +27,50 @@ def _initFileMenu(self):
         fileMenu = self._menu.addMenu('&File')
         fileMenu.addAction(self._createNewAction())
         fileMenu.addAction(self._createOpenAction())
+
+        lastOpened = fileMenu.addMenu("Open &Recent")
+        for ind in range(self.maxRecentFiles):
+            self.recentFileArray.append(QtWidgets.QAction(self, visible=False, triggered=self.openRecentFile))
+            lastOpened.addAction(self.recentFileArray[ind])
+
         fileMenu.addAction(self._createSaveAction())
         fileMenu.addAction(self._createRevertAction())
         fileMenu.addSeparator()
         fileMenu.addAction(self._createPreferenceAction())
         fileMenu.addSeparator()
         fileMenu.addAction(self._createExitAction())
 
+    def updateRecentFileActions(self):
+        recentFileNames = self.getRecentFileNames()
+        ind = 1
+        for i in range(len(recentFileNames)):
+            try:
+                exist = os.path.isfile(recentFileNames[i])
```

Comment:
do we really need to check if file exist? A user may want to put files at the same place but sometime not mount netwoprk drive or whatever...